# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/DocAccessibleChild.cpp
# Commit: 7b00a16ad21e
# Full Hash: 7b00a16ad21ee2b820b679199ae3806250d861e9
# Author: Nathan LaPre <nlapre@mozilla.com>
# Date: 2024-09-10 09:03:27
# Regressor Bug: 1794974
# File Overlap Count: 1
# Description:
#   Bug 1794974: Part 2: Add ability to query and send individual cache domains, r=Jamie
#   
#   This revision adds the capability of querying and sending information about
#   individual cache domains. It introduces the concept of active cache domains to
#   the accessibility service: cache domains that we think clients need. Conversely,
# ==============================================================================

diff -r 642b6d8c4d3b -r 7b00a16ad21e accessible/ipc/DocAccessibleChild.cpp
--- a/accessible/ipc/DocAccessibleChild.cpp	Mon Sep 09 23:02:19 2024 +0000
+++ b/accessible/ipc/DocAccessibleChild.cpp	Mon Sep 09 23:02:19 2024 +0000
@@ -60,8 +60,9 @@
   // Even though we send moves as a hide and a show, we don't want to
   // push the cache again for moves.
   if (!aAcc->Document()->IsAccessibleBeingMoved(aAcc)) {
-    fields =
-        aAcc->BundleFieldsForCache(CacheDomain::All, CacheUpdateType::Initial);
+    fields = aAcc->BundleFieldsForCache(
+        nsAccessibilityService::GetActiveCacheDomains(),
+        CacheUpdateType::Initial);
     if (fields->Count() == 0) {
       fields = nullptr;
     }